Makefile

```diff
@@ -185,6 +185,10 @@ lister-gen: ## Download lister-gen
 fmt: ## Run go fmt against code.
 	go fmt ./...
 
+.PHONY: resolve
+resolve: ## Run go mod tidy against code.
```

Comment:
could we try to get rid of these resolves down the line? I would like to avoid having to pin golang versions like we had to for the old reconciler